.circleci/config.yml

```diff
@@ -612,9 +621,10 @@ jobs:
                         name: '[FL] TestFlight Upload'
                         working_directory: native
             - run:
-                command: bundle exec fastlane ios sentry_upload version_name:${NEW_VERSION_NAME} version_code:${NEW_VERSION_CODE} build_config_name:<< parameters.build_config_name >> sourcemap_path:attached_workspace/<< parameters.build_config_name >>.ios.bundle.map
-                name: '[FL] Upload to Sentry'
-                working_directory: native
+                command: |
+                    eval $(yarn workspace --silent build-configs --silent manage to-bash << parameters.build_config_name >> ios)
+                    tools/sentry-release "$BUILD_CONFIG_BUNDLE_IDENTIFIER" "${NEW_VERSION_NAME}" ~/attached_workspace/sourcemaps/<< parameters.build_config_name >> "${NEW_VERSION_CODE}"
+                name: Sentry Upload
```

Comment:
Why did you change the naming of the command?

Reply:
Because it is no longer part of FastLane